marconi-chain-index/test/Spec/Marconi/ChainIndex/Indexers/Utxo/UtxoIndex.hs

```diff
@@ -404,6 +404,7 @@ propUtxoQueryShouldHaveSameDatumAsDatumQuery = Hedgehog.property $ do
   pure ()
 
 {- |
+ - TODO: test it in the event extraction logic, not in the indexer
```

Comment:
Did you want to do this in a separate PR?

Reply:
No, it was my notes to check which test I wanted to replicate or not in the new indexers, I'll remove them.